frontend/add-client/form-inputs/contact-information-inputs.component.tsx

```diff
@@ -160,8 +160,9 @@ export default React.forwardRef(function ContactInformationInputs(
 });
 
 function getTodayAsString() {
-  const date = new Date().toISOString();
-  return date.slice(0, date.indexOf("T"));
+  const dateFormat = require("dateformat");
```

Comment:
Same here -- we should use `import` instead of `require`. The documentation for dateformat assumes we're running in NodeJS, but we're not and we should use `import`

Reply:
done